pandas/util/testing.py

```diff
@@ -1324,11 +1323,13 @@ def assert_series_equal(left, right, check_dtype=True,
         # .values is an ndarray, but ._values is the ExtensionArray.
         # TODO: Use .array
         assert is_extension_array_dtype(right.dtype)
-        return assert_extension_array_equal(left._values, right._values)
+        assert_extension_array_equal(left._values, right._values)
+        return
 
     elif (is_extension_array_dtype(left) and not is_categorical_dtype(left) and
           is_extension_array_dtype(right) and not is_categorical_dtype(right)):
-        return assert_extension_array_equal(left.array, right.array)
+        assert_extension_array_equal(left.array, right.array)
+        return
```

Comment:
same don't need a return

Reply:
I tried this earlier but some of the tests failed so I added the returns back.

Reply:
See https://github.com/pandas-dev/pandas/pull/25462/commits/091a00e2aa5a7bb81a16731ad77ca11405e514d6 (this was in response to an earlier PR comment)